src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -837,7 +837,7 @@ private string ManifestComment(string insert, StreamWriter streamWriter)
         /// </summary>
         protected override void EndProcessing()
         {
-            // Win8: 264471 - Error message for New-ModuleManifest �ProcessorArchitecture is obsolete.
+            // Win8: 264471 - Error message for New-ModuleManifest - ProcessorArchitecture is obsolete.
```

Comment:
It was an en-dash character, so I figured it was meant to be a dash. If it's supposed to be a single-hyphen prefix, then that's what I'll put in there.